config/jobs/etcd/etcd-bbolt-periodics.yaml

```diff
@@ -0,0 +1,35 @@
+---
+periodics:
+- name: ci-bbolt-robustness-main-arm64
+  cron: "25 9 * * *" # runs every day at 09:25 UTC
+  cluster: k8s-infra-prow-build
+  extra_refs:
+    - org: etcd-io
+      repo: bbolt
+      base_ref: main
+  decorate: true
+  decoration_config:
+    timeout: 210m
+  annotations:
+    testgrid-dashboards: sig-etcd-bbolt-periodics
+    testgrid-tab-name: ci-bbolt-robustness-main-arm64
+  spec:
+    containers:
+      - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20241021-d3a4913879-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - |
+            apt-get -o APT::Update::Error-Mode=any update && apt-get --yes install dmsetup xfsprogs
+            ROBUSTNESS_TESTFLAGS="--count 100 --timeout 200m -failfast" make test-robustness
+        resources:
+          requests:
+            cpu: "4"
+            memory: "8Gi"
+          limits:
+            cpu: "4"
+            memory: "8Gi"
```

Comment:
Do the bbolt robustness tests require privileged like the etcd robustness tests do?

Reply:
Good question. And, I don't think so (at least not by reading the current workflow): https://github.com/etcd-io/bbolt/blob/12749bc979b4932a78d198f11a0f08f4ae1e3a23/.github/workflows/robustness_template.yaml. But may I be missing something obvious?

Reply:
Cool no lazyfs for bbolt requiring fuse so should be all good, thanks!